### DISCUSSION TITLE
How to: Copy an object with ClipSeg Masking to paste it into another image

### DISCUSSION THREAD

**Discussion by ntdviet** [WASasquatch/was-node-suite-comfyui#177] [Q&A] [answered]
Hi, can someone help me with a workflow that can do that?
Basically, I'd like to find a face, or an object, using ClipSeg Masking, than put a boundary around that mask and copy only that part of the image/latent to be pasted into another image/latent.

Is it possible using WAS pack? I still struggle to understand the application of all the nodes in there.
Thanks in advance.

**Comment by WASasquatch** [comment, marked as answer]
Mask Crop Region and then feed the top, left, right, and bottom coordinates to a Image Crop Location node. Then you can use the CROP_DATA output on a Image Paste node.

**Reply by ntdviet** [reply]
Ya thanks, I also saw it in your example, after digging through your md :)

Anyway last question @WASasquatch : I tried LatentCompositeMasked but it's always throwing an error with spatial dimensions mismatch. What am I doing wrong here?
![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/54492570/3206554f-d8a3-4a8b-a48f-64802adbd928)
![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/54492570/cc5b6708-4c8f-4f77-93c1-1c081ff44889)

**Reply by WASasquatch** [reply]
Latent composition needs same sized latenta and then you mask parts you want. Where mask is also same size.

**Reply by ntdviet** [reply]
Good to know, thanks. I've been using LatentComposite for variable latent sizes, so I thought it would be as convenient when a mask is involved. Ok, that's inconvenient but still manageable
